configuration/driver/icedrv_init.F90

```diff
@@ -1296,7 +1296,7 @@ subroutine set_state_var (nx, &
 
       use icedrv_arrays_column, only: hin_max
       use icedrv_domain_size, only: nilyr, nslyr, max_ntrcr, ncat, nfsd
-      use icedrv_arrays_column, only: floe_rad_c, floe_binwidth
+      use icepack_fsd, only: floe_rad_c, floe_binwidth
```

Comment:
This is NOT allowed.  If you need floe_rad_c and floe_binwidth in the driver, you need to get it via subroutine arguments, either a query to parameters or add it back into the icepack_init_fsd.  The values can be passed out to the driver AND still live inside Icepack so we don't have to keep passing them in.

Reply:
I'm adding back floe_rad_l, floe_rad_c, floe_binwidth, and c_fsd_range as optional "_out" arguments in icepack_init_fsd_bounds.  Any driver can then have them returned to them as part of that call and store them in the driver.  The fields will always be defined inside Icepack.